app/actions/actionTypes.js

```diff
@@ -0,0 +1,5 @@
+export const ADD_HIGHLIGHT = 'ADD_HIGHLIGHT';
```

Comment:
oops, I thought I'd completely reset to master...no, I don't mean to add these files.